packages/mgt-components/src/graph/graph.userWithPhoto.ts

```diff
@@ -61,7 +61,7 @@ export async function getUserWithPhoto(
   }
   if (getIsPhotosCacheEnabled()) {
     cachedPhoto = await getPhotoFromCache(userId || 'me', schemas.photos.stores.users);
-    if (cachedPhoto !== undefined && getPhotoInvalidationTime() > Date.now() - cachedPhoto.timeCached) {
+    if (cachedPhoto !== undefined && cachedPhoto !== null && getPhotoInvalidationTime() > Date.now() - cachedPhoto.timeCached) {
```

Comment:
Looks like this line might be too long for our prettier rules: https://github.com/microsoftgraph/microsoft-graph-toolkit/actions/runs/3431977122/jobs/5720739291#step:7:18

@devdavest Could you please fix this up and ensure that prettier is not failing.

Reply:
This should do the trick. It works fine on my local copy.